aws/table_aws_vpc_flow_log.go

```diff
@@ -0,0 +1,227 @@
+			{
+				Name:        "tags",
+				Description: resourceInterfaceDescription("tags"),
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromP(getVpcFlowlogTurbotData, "Tags"),
+			},
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromP(getVpcFlowlogTurbotData, "Title"),
+			},
+			{
+				Name:        "akas",
+				Description: resourceInterfaceDescription("akas"),
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getVpcFlowlogAkas,
+				Transform:   transform.FromValue(),
+			},
+		}),
+	}
+}
+
+//// LIST FUNCTION
+
+func listVpcFlowlogs(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+
+	// TODO put me in helper function
+	var region string
+	matrixRegion := plugin.GetMatrixItem(ctx)[matrixKeyRegion]
+	if matrixRegion != nil {
+		region = matrixRegion.(string)
+	}
+	plugin.Logger(ctx).Trace("listVpcFlowlogs", "AWS_REGION", region)
+
+	// Create session
+	svc, err := Ec2Service(ctx, d, region)
+	if err != nil {
+		return nil, err
+	}
+
+	err = svc.DescribeFlowLogsPages(
+		&ec2.DescribeFlowLogsInput{},
+		func(page *ec2.DescribeFlowLogsOutput, lastPage bool) bool {
+			for _, item := range page.FlowLogs {
+				d.StreamListItem(ctx, item)
+			}
+			return !lastPage
+		},
+	)
+
+	return nil, err
+}
+
+//// HYDRATE FUNCTIONS
+
+func getVpcFlowlog(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	logger := plugin.Logger(ctx)
+	logger.Trace("getVpcFlowlog")
+	quals := d.KeyColumnQuals
+	flowlogID := quals["flow_log_id"].GetStringValue()
+	// TODO put me in helper function
+	var region string
+	matrixRegion := plugin.GetMatrixItem(ctx)[matrixKeyRegion]
+	if matrixRegion != nil {
+		region = matrixRegion.(string)
+	}
+
+	// Create session
+	svc, err := Ec2Service(ctx, d, region)
+	if err != nil {
+		return nil, err
+	}
+
+	params := &ec2.DescribeFlowLogsInput{
+		FlowLogIds: []*string{&flowlogID},
+	}
+
+	//get call
+	item, err := svc.DescribeFlowLogs(params)
+	if err != nil {
+		logger.Debug("getVpcFlowlogs__", "Error", err)
+		return nil, err
+	}
+
+	if item.FlowLogs != nil && len(item.FlowLogs) > 0 {
+		return item.FlowLogs[0], nil
+	}
+
+	return nil, nil
+}
+
+func getVpcFlowlogAkas(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	plugin.Logger(ctx).Trace("getVpcFlowlogAkas")
+	vpcFlowlog := h.Item.(*ec2.FlowLog)
+	commonData, err := getCommonColumns(ctx, d, h)
+	if err != nil {
+		return nil, err
+	}
+	commonColumnData := commonData.(*awsCommonColumnData)
+
+	akas := []string{"arn:" + commonColumnData.Partition + ":ec2:" + commonColumnData.Region + ":" + commonColumnData.AccountId + ":flow-log/" + *vpcFlowlog.FlowLogId}
+
+	return akas, nil
+}
+
+// TRANSFORM FUNCTIONS
+
+func getVpcFlowlogTurbotData(_ context.Context, d *transform.TransformData) (interface{}, error) {
```

Comment:
Can we change the function name to `vpcFlowlogTurbotData`?
Generally we don't use `get` prefix for `TRANSFORM FUNCTIONS`.